cmd/root.go

```diff
@@ -66,34 +71,58 @@ var rootCmd = &cobra.Command{
 	},
 }
 
+// check if the version file exists in the container
+func checkVersionExists(versionUrl string) bool {
+	resp, err := http.Get(versionUrl)
+	if err != nil {
+		log.Err("checkVersionExists: error getting version file from container [%s]", err)
+		return false
+	}
+
+	return resp.StatusCode != 404
+}
+
+func getRemoteVersion(req string) (string, error) {
+	resp, err := http.Get(req)
```

Comment:
why did we choose to manually write the http request code instead of just using the GoSDK?

Reply:
we can use the sdk as well, but as this is happening before our pipeline starts we might not have all the config here. so creating a pipeline for gosdk doing some auth and all will be overkill. Thats what I feel, if there is an easier way to access public files in container easily using sdk then we shall explore that.